StarWars.Android/Views/MainActivity.cs

```diff
@@ -0,0 +1,27 @@
+using _Microsoft.Android.Resource.Designer;
+using MvvmCross.Platforms.Android.Presenters.Attributes;
+using MvvmCross.Platforms.Android.Views;
+using StarWars.Core.ViewModels;
+
+namespace StarWars.Android.Views;
+
+[MvxActivityPresentation]
+[Activity(Label = "@string/app_name", MainLauncher = true)]
+public class MainActivity : MvxActivity<MainViewModel>
+{
+    protected override void OnCreate(Bundle savedInstanceState)
+    {
+        base.OnCreate(savedInstanceState);
+        
+        SetContentView(ResourceConstant.Layout.activity_main);
+        
+        if (savedInstanceState == null)
+        {
+            var fragment = new CharactersFragment();
```

Comment:
if you use fragment here, you should create also fragment for list.
Activity should be only as container